### PR TITLE
Yx feature fonts

### PR DESCRIPTION
**Pull request by ybxiang05**
added custom fonts to screens
removed comments as necessary

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -41,5 +41,10 @@
   },
   "jest": {
     "preset": "react-native"
+  },
+  "rnpm": {
+    "assets": [
+      "./js/assets/fonts/"
+    ]
```
Comment (siddharthparmar7) [inline client/package.json]: No need to do this if you are using React Native >= 0.60

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Why are you adding pdf files to GitHub?

File: client/js/screens/Trips/Trips.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import {Text, TouchableHighlight} from 'react-native';
-import Itinerary from '../Itinerary';
 
 const Trips = props => {
   console.log(props);
```
Comment (iyuvraajsingh) [inline client/js/screens/Trips/Trips.js]: Please remove this console log.

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]

**Review by iyuvraajsingh — APPROVED** [review]
Good Work!!

**Pull request merged**